src/tests/helpers/passportLocal.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../app';
+import db from '../../models';
+import passport from '../../config/passportLocalConfig';
+import { User } from '../../queries';
+import * as validate from '../../helpers/validation';
+import status from '../../config/status';
+
+chai.should();
+
+let user = {};
+chai.use(chaiHttp);
+const newUser = {
+  firstName: 'senga',
+  lastName: 'mugabo',
+  username: 'daprince',
+  email: 'daprince@gmail.com'
+};
+
+describe('test locol passport authenticate', () => {
```

Comment:
This is a typo error

Reply:
this was the mistake, but I corrected it